src/components/Books.jsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import BookElement from './SingleBook';
 import AddBook from './AddBook';
 
```

Comment:
- [ ] `store` should be imported and the main React component `<App>` should be wrapped in _Provider_ and assign `store` in your _index.js_, similar to:

```ruby
    import store from './redux/configureStore';

    ReactDOM.render(
    <Provider store={store}>
        <App />
    </Provider>,
    document.getElementById('root')
    )
```